airbyte-integrations/connectors/destination-gcs/src/main/resources/spec.json

```diff
@@ -258,6 +258,12 @@
                 "type": "integer",
                 "default": 5,
                 "examples": [5]
+              },
+              "gzip_compression": {
```

Comment:
does it make sense to have this be compression_type, as a oneOf (none vs gzip)? Would make it easier in the future to add other compression algos.

Reply:
Will do.